### ISSUE TITLE
Support logging of non-standard (custom) conditions

### ISSUE THREAD

**Issue by aryoda** [aryoda/tryCatchLog#45]
Currently `tryCatchLog()` deliberately stops with an error if the catched condition is not of the type `error`, `warning` or `message`.

Custom condtions (self-made condition classes) are not yet supported unless they inherit from one of the above condition classes which would require user code changes for existing R code (which is against `tryCatchLog`s philosophy by wrapping existing code).

The user interrupt condition of R is most likely also not supported (see #14) currently.

Add support for this.

Basically all non-standard conditions must "just" be matched to a logging severity level...

**Comment by aryoda** [comment]
Related to #14

**Comment by aryoda** [comment]
Perhaps adding `interrupt` and (generic) `condition` support just to log those kind of conditions suffice...

**Comment by aryoda** [comment]
Implemented in v1.1.8
